.github/workflows/build.yml

```diff
@@ -199,6 +199,12 @@ jobs:
           exit 1
         fi
 
+        # Check that commit contains ':'
+        if git log --format=%s origin/"$BASE"..HEAD |  grep -v ':'; then
```

Comment:
Note: we will not report errors with the following kind of commits:
- `:smiley: hey, how are you`
- `: I'm a colon at the start... but I know that's not what you really want`
- `I'm a colon at the end... but I know that's not what you really want:`

I don't know how far we want to take this, probably not too far but I see a few ideas. We could at least enforce there is characters before and after the 'colon' (at least one character that is not a colon).
```
grep -v '[^:]+:[^:]+'
```
Another option would be to enforce the list of prefix word, for example, we only want this list of prefix:
- `front:`
- `editoast:`
- etc...

Reply:
I changed the script to use a regex which enforces our commit format